fi_gains_toolControl:
  topic: '/mixed_platform/platform_state'
  toolPos: 
    p_a: [0.0,0.0,0.0,2000.0,800.0] # assymetric p gains, to make it easier to move e.g. in abduction w.r.t adduction
    p_b: [0.0,0.0,0.0,2000.0,1000.0]
    d: [10.0,10.0,100.0,60.0,60.0]
  toolSpeed:
    p_a: [100.0,100.0,3000.0,5000.0,800.0]
    p_b: [250.0,250.0,10000.0,5000.0,2000.0]
    d: [10.0,10.0,100.0,60.0,60.0]
  toolMixed:  # the auxiliary platform
    p_a: [100.0,100.0,0.0,10000.0,800.0]
    p_b: [250.0,250.0,0.0,10000.0,1000.0]
    d: [10.0,10.0,60.0,60.0,60.0]